Automattic/jetpack
# VideoPress: Use Blog ID in links to WPCOM instead of site slug

## Proposed changes:
Adjust checkout link in VideoPress package to use blog ID instead of slug for multi-URL support.

### Other information:


## Jetpack product discussion
pf5801-93-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
To test the blog, you'll need two domains pointing to the same site. One way to do that is to setup two proxying tunnels to the same local environment.

**Connect Jetpack with a free plan on URL A, and load it via URL B.**

If your site falls into IDC due to concurrency issues with IDC secret validation (fix pending), remove `jetpack_sync_error_idc` option.

1. Activate "Jetpack VideoPress" plugin.
2. Using URL B (not cache site main URL), go to "Jetpack -> VideoPress" and click the upgrade link. Confirm you're redirected to the proper checkout with URL A (cache site main URL) in the page URL.
3. Purchase the plan. Confirm you get redirected to URL A's "Jetpack -> VideoPress" page.
4. Go back to URL B's "Jetpack -> VideoPress" page. Confirm the plan has been upgraded.
